tests/test_cli.py

```diff
@@ -0,0 +1,58 @@
+from pathlib import Path
+import pysam
+import pytest
+from xopen import xopen
+
+from blr.cli.init import init
+from blr.cli.run import run
+
+TESTDATA_READS = Path("testdata/reads.1.fastq.gz")
+
+
+def count_bam_alignments(path):
+    with pysam.AlignmentFile(path) as af:
+        n = 0
+        for _ in af:
+            n += 1
+    return n
+
+
+def count_fastq_reads(path):
+    with xopen(path) as f:
+        n = 0
+        for _ in f:
+            n += 1
+    return n // 4
```

Comment:
This expression will floor the result to the closest integer. Maybe we could also include a check so we know it has a root four? I'm thinking 
```suggestion
    return n // 4 if (n % 4) == 0 else error
```

```
>>> 10 // 4
2
>>> 11 // 4
2
>>> 12 // 4
3
```

Reply:
(but something meaningful instead of where I just wrote ´error´)

Reply:
I don’t think it’s too important to check that here because the FASTQ file will be run the pipeline anyway where other tools will check it. It was just a way to avoid hardcoding the number of reads in the test data FASTQ. But I agree, it doesn’t hurt to do a check here and there’s a risk someone might take the function and use it somewhere else where the input is not known beforehand.

I think I’ll just parse the file with `dnaio` here, which will even check whether it’s a correctly formatted FASTQ file.

Reply:
Yeah that sounds like a good idea, that means it will be less complex as well since we're not introducing several kinds of error handling for the same thing.